i18n/locales/ca.json

```diff
@@ -1,39 +1,40 @@
 {
-    "components.footer.scrollToTop.button": "Torna al començament",
-    "components.header.links.home": "Inici",
-    "components.header.links.about": "Quant a",
-    "components.header.links.download": "Descàrregues",
-    "components.header.links.docs": "Documentació",
-    "components.header.links.getInvolved": "Participa",
-    "components.header.links.security": "Seguretat",
-    "components.header.links.certification": "Certification",
-    "components.header.links.blog": "Notícies",
-    "components.navigation.about.links.governance": "Govern",
-    "components.navigation.docs.links.es6": "ES6 i més enllà",
-    "components.navigation.docs.links.apiLts": "{fullLtsNodeVersion} API {spanLts}",
-    "components.navigation.docs.links.apiCurrent": "{fullCurrentNodeVersion} API",
-    "components.navigation.docs.links.guides": "Guies",
-    "components.navigation.docs.links.dependencies": "Dependències",
-    "components.navigation.getInvolved.links.collabSummit": "Col·labora en el Summit",
-    "components.navigation.getInvolved.links.contribute": "Contribueix",
-    "components.navigation.getInvolved.links.codeOfConduct": "Codi de Conducta",
-    "components.downloadList.links.previousReleases": "Descarrègues",
-    "components.downloadList.links.packageManager": "Instal·lar Node.js mitjançant un gestor de paquets",
-    "components.downloadList.links.shaSums": "Signatures SHASUMS per arxius de versions",
-    "components.downloadList.links.shaSums.howToVerify": " (Com verificar-ho)",
-    "components.downloadList.links.allDownloads": "Totes les opcions de descàrrega",
-    "components.downloadList.links.nightlyReleases": "Versions Nightly",
-    "components.downloadList.links.unofficialBuilds": "Unofficial builds",
-    "components.downloadList.links.buildingFromSource": "Building Node.js from source on supported platforms",
-    "components.downloadList.links.installingOnLinux": "Installing Node.js via binary archive",
-    "components.downloadList.links.installingOnWsl": "Install on Windows Subsystem for Linux (WSL)",
-    "components.downloadReleasesTable.changelog": "Registre de Canvis",
-    "components.downloadReleasesTable.releases": "Releases",
-    "components.downloadReleasesTable.docs": "Documentació",
-    "components.header.buttons.toggleLanguage": "Toggle Language",
-    "components.header.buttons.toggleDarkMode": "Toggle dark/light mode",
-    "components.pagination.next": "Següent | ",
-    "components.pagination.previous": "Anterior",
-    "layouts.blogPost.author.byLine": "{author, select, null {} other {By {author}, }}",
-    "layouts.blogIndex.currentYear": "News from {year}"
-  }
+  "components.footer.scrollToTop.button": "Torna al començament",
```

Comment:
Any reason why git thinks the whole file got changed?

Reply:
There was an indentation issue here